doc/cascadia/profiles.schema.json

```diff
@@ -2376,6 +2376,11 @@
           "description": "When set to true, the terminal will focus the pane on mouse hover.",
           "type": "boolean"
         },
+        "disableMouseZoom": {
```

Comment:
Here I actually don't mind using "zoom" since it's less verbose. I still think we should avoid the double negative here though (see the other comment in MTSMSettings.h).